src/orejime.js

```diff
@@ -48,7 +48,7 @@ export const defaultConfig = {
     stringifyCookie: JSON.stringify.bind(JSON),
     parseCookie: JSON.parse.bind(JSON),
     privacyPolicy: '',
-    default: true,
+    default: false,
```

Comment:
I don't think this should be part of the changes. 
To me, it is another concern, it is only affecting whether the apps are on or off by default within the advanced settings.

Reply:
this is been a while, i don't remember everything haha ^^ effectively actually it doesn't seem like the best way to do that

Reply:
Thank you!